config/locales/en/images/crop.yml

```diff
@@ -3,3 +3,4 @@ en:
     crop:
       title: "Crop image for ‘%{title}’"
```

Comment:
@benhazell do we need all 3 of these titles?

Reply:
I didn't realise the crop screen in the lead image flow had a different title.
I think the lead image crop and the insert image crop should be the same. Then we only need 2 titles - one for cropping the image, and one for if the image is the exact size and can't be cropped.

Reply:
@benthorner updated to get rid of the redundant modal title